contracts/contracts/strategies/balancer/BaseBalancerStrategy.sol

```diff
@@ -375,11 +390,18 @@ abstract contract BaseBalancerStrategy is InitializableAbstractStrategy {
         internal
         returns (uint256 unwrappedAmount)
     {
-        if (asset == stETH) {
+        if (asset == stETH && amount > 0) {
```

Comment:
nit: I feel like this could changed to an nested `if` structure to not evaluate `amount > 0` multiple times (but the gas cost saved might be very insignificant):
```
if (amount > 0 ) {
    if (asset == stETH) {
     ...
     } else if (asset == frxETH) {
     ...
     }
}

Reply:
Yup good point. I've also though about that and decided to go for a little more clarity with an arguably worse gas consumption.